### PR TITLE
Git api

### PR DESCRIPTION
**Pull request by dskuzm**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,12 +11,13 @@
     <div class="holder">
         <h1>search</h1>
         <div class="input_holder">
-            <input type="text" placeholder="enter your text" value="js" id="search">
```
Comment (ashatova) [inline index.html]: Change ID to data-attr

**Comment by ashatova** [comment]
Good work!

File: js/script.js [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
-    xmlHttp.open("GET", "https://api.github.com/search/repositories?q=$test", true);
-
-    xmlHttp.onreadystatechange = function (oEvent) {
-        if (xmlHttp.readyState === 4) {
-            if (xmlHttp.status === 200) {
-                var obj = JSON.parse(xmlHttp.responseText);
-                console.log(obj);
-            } else {
-                console.log('Error', xmlHttp.statusText);
+        xmlHttp.onreadystatechange = function (oEvent) {
```
[the remark this answers is not in the document]
Comment (dskuzm) [reply js/script.js]: done

File: js/script.js [hunk 3]
```diff
@@ -1,27 +1,26 @@
...
+            if (xmlHttp.readyState === 4) {
+                if (xmlHttp.status === 200) {
+                    var obj = JSON.parse(xmlHttp.responseText);
```
[the remark this answers is not in the document]
Comment (dskuzm) [reply js/script.js]: done

File: index.html [hunk 4]
```diff
@@ -11,12 +11,13 @@
...
-            <input type="submit">
+            <input type="text" placeholder="enter your text" id="search">
+            <input type="submit" id="submit">
         </div>
 
         <div class="result_holder">
-            <span>your result</span>
+            <h2>Search result</h2>
+            <a id="repoLink">no result</a>
```
[the remark this answers is not in the document]
Comment (dskuzm) [reply index.html]: done

File: js/script.js [hunk 5]
```diff
@@ -1,20 +1,19 @@
...
+        xmlHttp.open('GET', 'https://api.github.com/search/repositories?q=$' + inputText, true);
     
-        xmlHttp.onreadystatechange = function (oEvent) {
+        xmlHttp.onload = function (oEvent) {
             if (xmlHttp.readyState === 4) {
                 if (xmlHttp.status === 200) {
-                    var obj = JSON.parse(xmlHttp.responseText);
-                        var link = (obj.items[6].html_url);//test code
+                    var data = JSON.parse(xmlHttp.responseText);
+                        var link = (data.items[6].html_url);
```
Comment (ashatova) [inline js/script.js]: why 6 element?

**Pull request merged**